### ISSUE TITLE
Alpha channel support?

### ISSUE THREAD

**Issue by leilei-** [WL-Amigo/waifu2x-converter-cpp#3]
Could it process the PNG's alpha channel and process it separately from the color channels?

Related issue: https://github.com/nagadomi/waifu2x/issues/27

**Comment by RyanBram** [comment]
Yeah, please add Alpha Channel support as it will be very useful.

**Comment by robotortoise** [comment]
I know this is old, but I would really appreciate alpha channel support as well. waifu2x would be fantastic for texture hacking and upscaling!

**Comment by amunozj** [comment]
I would like to add my voice here.   Alpha channel support would be so awesome!
